app/src/main/java/com/x8bit/bitwarden/BitwardenLegacyAppComponents.kt

```diff
@@ -0,0 +1,39 @@
+package com.x8bit.bitwarden
+
+/**
+ * Contains all legacy component names. Theses names do not match the current naming conventions
+ * but must be used in the AndroidManifest to properly maintain compatibility.
+ */
+data object BitwardenLegacyAppComponents {
```

Comment:
Does this wrapper need to exist? Can we just keep the constants as top-level values that exist together in this file?